User-Help/GlobalTrustLevels.md

```diff
@@ -0,0 +1,133 @@
+
+
+
+# Glossary of terms
+
+### Prerequisites
+
+Minimum activity required to reach a given trust level.
+
+### Posts
+
+Posts are the questions and answers, blogs and articles that compose the core of Codidact. Comments are not considered posts.
+
+### Responses 
+
+A response is a standalone reply to a post, typically a complete and well-informed answer to a question. It may be something else depending on category, e.g. a submission to a contest, but its intent is distinctly different from a comment.
+
+### Comments
+
+A comment, in comparison to a response, is a short note referring to either a post, response or another comment. It is usually used for clarification or to request more information.
+
+### Flags
+
+Flags send a user contribution to a moderation queue for review. Use these to mark obsolete comments or answers, material inappropriate for a particular site, or anything that breaks the Code of Conduct or Terms of Service. 
+
+### Upvotes
+
+Upvotes help a post or answer gain visibility and act as an indicator of how the community values the contribution. They indicate that a question is useful, well-written, and helpful to other users of the site. They are also used in combination with downvotes to determine if it is well-received. 
+
+### Suggest edits
+
+Sometimes a contribution could use a little help. This could be due to grammar and formatting issues, or maybe the original poster has mentioned important information in a comment and needs to add it to the original post. Suggesting edits helps improve the quality of a site's content.
+
+### Suggest duplicates
+
+You may not have been the first person to ask this question! When that happens, adding a link to to the prior post helps you and everyone else find the answers more quickly. 
+
+### Downvotes
+
+Downvotes are the negative analogue to upvotes. They are used to reduce the visibility of contributions with low value to the community. Often they signal that a post is hard to understand or incomplete, or that an answer is wrong or off-topic. They are also used in the calculation of whether or not a particular contribution is well-received. 
+
+### Vote-to-hold
+
+Some questions might be off-topic, or unclear without additional information from the original poster. In these cases it's best to put a temporary hold on response until the original poster updates his work. All holds will come with an explanation, and will be removed once the necessary edits are made.
```

Comment:
Missed a pronoun. :-)  How about "updates the question"?

Reply:
Doh!